--- _video.py.bak	2026-01-24 12:32:21.063741168 +0000
+++ _video.py	2026-01-24 12:32:52.144758731 +0000
@@ -3308,8 +3308,8 @@
                 proto = 'https'
                 https_fmts = []
                 for fmt_stream in streaming_formats:
-                    if fmt_stream.get('targetDurationSec'):
-                        continue
+                    #if fmt_stream.get('targetDurationSec'):
+                        #continue

                     # FORMAT_STREAM_TYPE_OTF(otf=1) requires downloading the init fragment
                     # (adding `&sq=0` to the URL) and parsing emsg box to determine the
